### ISSUE TITLE
"Last Updated" text needs more context

### ISSUE THREAD

**Issue by sunahsuh** [mozilla/redash#810]
### Issue Summary

"Last Updated" text needs more context — there are two places on the query view that say "Updated XX ago," one for presumably the data and one for, I assume, the query itself? (although I didn't edit the query yesterday so I'm unsure what it actually refers to.) There's no real way to know what each of those pieces of text are referring to.
![screenshot_2019-02-07 percentage of duplicates in main summary](https://user-images.githubusercontent.com/953153/52437395-6bb97a00-2adc-11e9-8109-5caf03897c2d.png)

### Steps to Reproduce

1. This is on every saved query page, but here's the example I was looking at today: https://sql.telemetry.mozilla.org/queries/61069

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

### Technical details:

* Redash Version:
* Browser/OS:
* How did you install Redash:

**Comment by jezdez** [comment]
Yeah, definitely something that we should consider, but better file a ticket upstream.
